cluster-scope/bundles/curator-operator-system/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base/core/namespaces/curator-operator-system
+  - ../../base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-manager-rb
+  - ../../base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-oauth-proxy-rb
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-auth-proxy-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-manager-role
+  - ../../base/rbac.authorization.k8s.io/clusterroles/curator-operator-metrics-reader
```

Comment:
Is this clusterrole bound to something that I missed out?

Reply:
Roles binded to role bindings which is bound to curator-operator-controller-manager service account.

Reply:
I am asking specifically about the clusterrole `curator-operator-metrics-reader`. Is that bound to something because I do not see a clusterrolebinding that binds that role to the `curator-operator-controller-manager` service account? Could you point me to that rolebinding if it exists?

Reply:
That's not bound to anything. If it's not bound to anything. That means it's not applied and even if it's removed will or should not impact the code. I am sure there is no role binding for this role.

Reply:
Removed and pushed the commit.